### ISSUE TITLE
Cannot process confirmed transaction for unknown subchannel

### ISSUE THREAD

**Issue by holzeis** [get10101/10101#2189]
Environment: test

Just got printed in my app logs during testing.. Channel was closed on the latest version

```
flutter: [INFO] TIME: 2024-03-08T08:53:08.717435Z r: dlc_manager::sub_channel_manager: Transaction 0409f0a654e8073e036dc791c7bac82c0fa5bde82bfca8c222cf667b6d5138c1 confirmed for channel 99e5fe5056efa4f49d8a34ca5522e587fd65ebe71de7b96e8358554affe7e351. Type: CollaborativeClose
flutter: [ERROR] TIME: 2024-03-08T08:53:08.717637Z r: dlc_manager::sub_channel_manager: Cannot process confirmed transaction 0409f0a654e8073e036dc791c7bac82c0fa5bde82bfca8c222cf667b6d5138c1 for unknown subchannel 99e5fe5056efa4f49d8a34ca5522e587fd65ebe71de7b96e8358554affe7e351
flutter: [DEBUG] TIME: 2024-03-08T08:53:08.723791Z r: dlc_manager::chain_monitor: Cleaning up data related to channel [153, 229, 254, 80, 86, 239, 164, 244, 157, 138, 52, 202, 85, 34, 229, 135, 253, 101, 235, 231, 29, 231, 185, 110, 131, 88, 85, 74, 255, 231, 227, 81]
```

**Comment by holzeis** [comment]
This error message seems to be harmless, since we don't use sub channels anymore. The processing is continued and the `CollaborativeClose` transaction type is processed on the dlc manager.

**Comment by holzeis** [comment]
Will be fixed with #1789
